R/unit.conversions.R

```diff
@@ -217,6 +221,7 @@ validMetadataUnits <- function(unitstr, unit.type=c("ANY","flow.units","conc.uni
 #' loadflex:::translateFreeformToUnitted("mg L^-1") # "mg L^-1"
 translateFreeformToUnitted <- function(freeform.units, attach.units=FALSE) {
   # Quick escape if our work is already done.
+  old=NULL
```

Comment:
this is removed https://github.com/miguelcleon/loadflex/blob/master/R/unit.conversions.R#L178